style.css

```diff
@@ -61,6 +61,7 @@ header {
 .close-icon:active {
   box-shadow: none;
   background-color: transparent;
+  cursor: pointer;
 }
 
```

Comment:
**Kindly update your ``README`` file.**
- Fill the missing information:
<img width="1440" alt="Screen Shot 2021-11-13 at 3 21 31" src="https://user-images.githubusercontent.com/22258274/141613542-d4dc7bdd-ccf3-4f44-913b-afd1286afa26.png">

Reply:
- Add a ``License`` to your project.

Reply:
Hello! I'm sorry, what missing information are we talking about? In previous code reviews I was told to eliminate all the unnecessary steps so I've done it.